apps/src/signUpFlow/FinishStudentAccount.tsx

```diff
@@ -186,9 +186,12 @@ const FinishStudentAccount: React.FunctionComponent<{
   };
 
   const submitStudentAccount = async () => {
+    if (!!isSubmitting) {
```

Comment:
Nit: since `isSubmitting` is set to `false` on load, I think this can just be `isSubmitting`

Reply:
You right you right. updated!